### PR TITLE
feat: Add getMaxPriorityFeePerGas method

### LINKED ISSUE #4195: Plans to support eth_maxPriorityFeePerGas method?
I've gone through the EIP 1559 PR and could not find support for this new method. There is a separate PR for eth_feeHistory, which is fantastic. What is the reasoning behind ignoring eth_maxPriorityFeePerGas? Your hard work is appreciated!

### PR DESCRIPTION
**Pull request by Muhammad-Altabba**
## Description

Fixes #4195

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

**Review by avkos — APPROVED** [review]

**Comment by luu-alex** [comment]
LGTM, is there any chance to create an intergration test as well

File: packages/web3-rpc-methods/test/unit/eth_rpc_methods/get_maxPriorityFeePerGas.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	let requestManager: Web3RequestManager;
+
+	beforeAll(() => {
+		requestManager = new Web3RequestManager('http://127.0.0.1:8545');
+		requestManagerSendSpy = jest.fn();
+		requestManager.send = requestManagerSendSpy;
+	});
+
+	it('should call requestManager.send with getMaxPriorityFeePerGas method', async () => {
+		await ethRpcMethods.getMaxPriorityFeePerGas(requestManager);
```
Comment (jdevcs) [inline packages/web3-rpc-methods/test/unit/eth_rpc_methods/get_maxPriorityFeePerGas.ts:46]: plus there should be e2e test for it before merging. Thanks @Muhammad-Altabba

Comment (jdevcs) [reply packages/web3-rpc-methods/test/unit/eth_rpc_methods/get_maxPriorityFeePerGas.ts:46]: in eth package

**Review by jdevcs — APPROVED** [review]

**Review by CharlesCowan — COMMENTED** [review]
[```

```]()

**Pull request merged**